### PR TITLE
Minor tweaks and fix for XFS

### PR DESCRIPTION
**Pull request by scottgerring**
Hi @mavi888,

A colleague and I have run through the workshop content and made a couple tweaks to update it for changes in Cloud9 and the JS bits. Let me know what you think, and thanks for the great content!

Scott

### CODE REVIEW

File: docs/add-api.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ amplify add api
 
 8. Select "Yes" when asked if you want to edit the schema now.
 
-Copy this schema in the schema.graphql file that pops up.
+If you aren't using Cloud9, copy this schema in the schema.graphql file that pops up and save it.
+If you are using Cloud9, the file won't open itself. You can find it by browsing to "amplify --> backend --> api --> notesapp"  in the directory explorer on the left and double-clicking on it, and then pasting the schema from below into it, and saving it.
```
Comment (scottgerring) [inline docs/add-api.md:28]: Add a note to make it clear that in Cloud9 the file won't open itself

File: docs/add-api.md [hunk 2]
```diff
@@ -72,7 +73,7 @@ Please manually edit the file created at /notes-app/amplify/backend/api/notesapp
 ? Press enter to continue
 ```
 
-9. Then push the changes to the cloud. Accept all the defaults and continue.
+9. Then push the changes to the cloud. Accept all the defaults and continue. This process will create the API in the cloud, and generate the code in your project you need to use it.
```
Comment (scottgerring) [inline docs/add-api.md:76]: To reinforce that all the somewhat-intimidating looking defaults are doing something we want !

File: docs/deploy-app.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ For this demo I will show you the local deployment. But feel free to investigate
 
 2. Then choose the option that says "Amazon CloudFront and S3"
 
-3. Then choose the option "DEV". As we are doing this for a demo < TODO MAKE SURE THAT THIS WORKS ON EVENTENGINE>
+3. Then choose the option "DEV". As we are doing this for a demo
```
Comment (scottgerring) [inline docs/deploy-app.md:26]: I ran this through in eventengine - it all works :)

File: docs/getting-started.md [hunk 4]
```diff
@@ -18,7 +18,10 @@ In this workshop, we’ll build an app with quite a few features, including:
 
 Here’s a map of the services we’ll use and how they’ll all connect.
 
-< TODO >
+<img src="../images/architecture.png"
```
Comment (scottgerring) [inline docs/getting-started.md:21]: Did a quick diagram to knock out your TODO - it is using draw.io and the diagram source itself is embedded in the image, so it should be easy to update/edit/change in the future.

File: docs/getting-started.md [hunk 5]
```diff
@@ -95,7 +100,7 @@ If you are not inside the notes-app directory, please go in there
 
 ```
 cd notes-app
-npm install bootstrap
+npm install bootstrap@4.5.2
```
Comment (scottgerring) [inline docs/getting-started.md:103]: Bootstrap's rolled forward a major version, which breaks some of the styles; forcing back to 4.5 fixes it again

File: docs/initialize-app.md [hunk 6]
```diff
@@ -16,7 +16,9 @@ amplify init
 
 2. Press Enter to accept the default project name ‘notesapp’
 
-3. Prese enter for the default environment name 'dev'
+3. Select 'n' when asked to 'Initialize the project with the above configuration'
```
Comment (scottgerring) [inline docs/initialize-app.md:19]: Looks like the UI flow for this has changed slightly - you need to say 'no' to this new prompt to get to the bit where you can fill the rest you need in

File: docs/prerequisites.md [hunk 7]
```diff
@@ -48,7 +48,9 @@ AWS Cloud9 is a cloud-based integrated development environment (IDE) that lets y
 
 5. Name it "workshop", and go to the Next step.
 
-6. Select Create a new instance for environment (EC2) and click _Other instance type_ and pick "t2.medium"
+6. Select Create a new instance for environment (EC2) and click _Other instance type_ and pick "t3.medium"
+
+7. Select **Amazon Linux 2** if it is not already selected
```
Comment (scottgerring) [inline docs/prerequisites.md:53]: This is the new default, and brings in the root-is-now-XFS change, which also means the resize.sh has to change a bit

File: utils/resize.sh [hunk 8]
```diff
@@ -29,14 +29,11 @@ if [ $(readlink -f /dev/xvda) = "/dev/xvda" ]
...
-  # Expand the size of the file system.
-  sudo resize2fs /dev/xvda1
-
 else
   # Rewrite the partition table so that the partition takes up all the space that it can.
   sudo growpart /dev/nvme0n1 1
+fi
+
+# Expand the size of the file system
+sudo xfs_growfs /
```
Comment (scottgerring) [inline utils/resize.sh:38]: Need to use this instead of resize2fs now that root is on XFS in the newer images

**Comment by mavi888** [comment]
Awsome! Thanks

**Pull request merged**